common/src/main/java/com/microsoft/identity/common/internal/activebrokerdiscovery/AccountManagerBrokerDiscoveryUtil.kt

```diff
@@ -50,7 +50,9 @@ class AccountManagerBrokerDiscoveryUtil(
     }
 
     constructor(context: Context): this(
-        knownBrokerApps = BrokerData.getKnownBrokerApps(),
+        knownBrokerApps = BrokerData.getKnownBrokerApps().filter {
+            BrokerData.isAccountManagerSupported(it.packageName)
+        }.toSet(),
```

Comment:
I think we are making this change to unblock Yixi for an issue with identifiying PROD LTW? But, with this line, it means that we are excluding LTW from knownBrokerApps right? 
How is this going to help with her testing?

Reply:
we don't want LTW to be included in the accountmanager based discovery.

this should not block her.